content/develop/data-types/timeseries/_index.md

```diff
@@ -19,45 +19,661 @@ weight: 150
-* Compaction for automatically updated aggregated timeseries
-* Secondary indexing for time series entries. Each time series has labels (field value pairs) which will allows to query by labels
+## Create a time series
+
+You can create a new empty time series with the [`TS.CREATE`]({{< relref "commands/ts.create/" >}})
+command, specifying a key name. Alternatively, if you use [`TS.ADD`]({{< relref "commands/ts.add/" >}})
+to add data to a time series key that does not exist, it is automatically created (see
+[Adding data points](#adding-data-points) below for more information about `TS.ADD`).
+
+```bash
+> TS.CREATE thermometer:1
+OK
+> TYPE thermometer:1
+TSDB-TYPE
+> TS.INFO thermometer:1
+ 1) totalSamples
+ 2) (integer) 0
+    .
+    .
+```
+
+The timestamp for each data point is a 64-bit integer value. This is designed
+to support Unix timestamps, measured in milliseconds since the
+[Unix epoch](https://en.wikipedia.org/wiki/Unix_time). However, you can interpret
+the timestamps in any way you like (for example, as the number of days since a given start date).
+When you create a time series, you can specify a maximum retention period for the
+data, relative to the last reported timestamp. A retention period of zero means
+the data does not expire.
+
+```bash
+# Create a new time series with a first value of 10.8 (Celsius),
+# recorded on day 1, with a retention period of 100 days.
+> TS.ADD thermometer:2 1 10.8 RETENTION 100
+(integer) 1
+> TS.INFO thermometer:2
+    .
+    .
+ 9) retentionTime
+10) (integer) 100
+    .
+    .
+```
+
+You can also add one or more *labels* to a time series when you create it. Labels
+are key-value pairs where the value can be a string or a number. You can use
+both the keys and values to select subsets of all the available time series
+for queries and aggregations.
+
+```bash
+> TS.ADD thermometer:3 1 10.4 LABELS location UK type Mercury
+(integer) 1
+> TS.INFO thermometer:3
+ 1) totalSamples
+ 2) (integer) 1
+ 3) memoryUsage
+ 4) (integer) 5000
+    .
+    .
+19) labels
+20) 1) 1) "location"
+       2) "UK"
+    2) 1) "type"
+       2) "Mercury"
+    .
+    .
+```
+
+## Add data points
+
+You can add individual data points with [`TS.ADD`]({{< relref "commands/ts.add/" >}}),
+but you can also use [`TS.MADD`]({{< relref "commands/ts.madd/" >}}) to add multiple data
+points to one or more time series in a single command. (Note that unlike `TS.ADD`, `TS.MADD`
+doesn't create any new time series if you specify keys that don't exist.) The return value
+is an array containing the number of samples in each time series after the operation.
+If you use the `*` character as the timestamp, Redis will record the current
+Unix time, as reported by the server's clock.
+
+```bash
+> TS.MADD thermometer:1 1 9.2 thermometer:1 2 9.9 thermometer:2 2 10.3
+1) (integer) 1
+2) (integer) 2
+3) (integer) 2
+```
+
+## Query data points
+
+Use [`TS.GET`]({{< relref "commands/ts.get/" >}}) to retrieve the last data point
```

Comment:
Maybe 'latest' instead of 'last'. It is not the last point added, but the point with the highest timestamp.